test-project/build.gradle

```diff
@@ -39,5 +39,8 @@ subprojects {
         testImplementation 'org.junit.jupiter:junit-jupiter-params:5.2.0'
         testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.2.0'
     }
-    
+
+    javadoc {
```

Comment:
I wonder if we should make `build`  depend on `javadoc` so we don't forget to trigger this build target from the command line, or when writing new tests.

Reply:
what's the right way to do that?